internal/datastore/spanner/spanner.go

```diff
@@ -116,7 +116,16 @@ func NewSpannerDatastore(database string, opts ...Option) (datastore.Datastore,
 		return nil, fmt.Errorf("failed to enable spanner GFE latency stats: %w", err)
 	}
 
-	client, err := spanner.NewClient(context.Background(), database,
+	client, err := spanner.NewClientWithConfig(context.Background(), database,
+		spanner.ClientConfig{
+			// obtained from DefaultSessionPoolConfig, only modified the min opened
+			SessionPoolConfig: spanner.SessionPoolConfig{
+				MinOpened:           config.minSessions,
+				MaxOpened:           config.maxSessions,
+				HealthCheckWorkers:  10,
+				HealthCheckInterval: 50 * time.Minute,
+			},
+		},
```

Comment:
I think we should copy the spanner.DefaultSesssionPoolConfig and change only the values we need.